src/ripple/app/misc/NetworkOPs.cpp

```diff
@@ -2121,9 +2130,16 @@ Json::Value NetworkOPsImp::getServerInfo (bool human, bool admin)
     }
     else
     {
-        info[jss::load_factor] =
-            static_cast<double> (app_.getFeeTrack ().getLoadFactor ()) /
-                app_.getFeeTrack ().getLoadBase ();
+        auto const loadBase = app_.getFeeTrack().getLoadBase();
+        assert(!escalationMetrics ||
+            loadBase == escalationMetrics->referenceFeeLevel);
+
+        info[jss::load_factor] = static_cast<double> (loadFactor) / loadBase;
```

Comment:
Yeah, this function is reused for both `server_info` and `server_state`, so the differences can cause a bit of confusion.

Unfortunately the different scaling factors came about because of the mixed units, as you've observed. The values are the same, so it's not like we're completely mixing apples and pianos. 

What I can do is a quick unit conversion before reporting `load_factor` so that if those values ever do change, it'll continue to "just work".